projects/1.20/assets/productivebees/productivebees/lang/zh_cn.json

```
@@ -0,0 +1,1015 @@
+  "productivebees.ingredient.description.collector_bee": "如果你把东西放在地上，这只蜜蜂会把它们收集起来并储存在蜂箱里。",
+  "productivebees.ingredient.description.creeper_bee": "爆炸！当它生气的时候\n⋯⋯也可能是节食的时候。\n\n你应该把它单独隔离起来。",
+  "productivebees.ingredient.description.cupid_bee": "这个蜜蜂分泌的信息素让动物们心情愉悦。",
+  "productivebees.ingredient.description.dark_oak_wood_nest": "这个蜂巢将吸引所有主世界生物群系中的蜜蜂。",
+  "productivebees.ingredient.description.digger_bee": "这种蜜蜂比较孤僻，它一般都独自生活在主世界的蜂巢里。查阅关于它的合成表来看看他会在哪些蜂巢中生成。",
+  "productivebees.ingredient.description.dragon_egg_hive": "一种为末影龙蜜蜂设计的特殊蜂箱，使得使用者可以收获到原本难以获得的末影龙蜜脾。\n生活在末地的龙蛋蜂巢中的末影龙蜜蜂不需要采蜜，因为它们会从空气中吸收到末影龙的精华。当在任意其他蜂巢工作时，末影龙蜜蜂就必须有另一个授粉来源。",
+  "productivebees.ingredient.description.dye_bee": "带回来的不仅仅有花蜜，还有花瓣碎片。",
+  "productivebees.ingredient.description.end_stone_nest": "这个蜂巢会吸引末地的蜜蜂。为了吸引蜜蜂，蜂巢必须放在末地。",
+  "productivebees.ingredient.description.farmer_bee": "庄稼收获后留下一片狼藉。为了保持农场清洁，你可能想投资一只囤积者蜜蜂。",
+  "productivebees.ingredient.description.glowstone_nest": "吸引下界的发光蜜蜂。为了吸引蜜蜂，蜂巢必须放在下界。",
+  "productivebees.ingredient.description.gravel_nest": "原生于温暖河流和沙滩的蜜蜂会被吸引到这个蜂巢。蜂巢必须放置在河流或沙滩生物群系中以吸引蜜蜂。",
+  "productivebees.ingredient.description.green_carpenter_bee": "这种蜜蜂比较孤僻，它一般都独自生活在主世界的蜂巢里。查阅关于它的合成表来看看他会在哪些蜂巢中生成。",
+  "productivebees.ingredient.description.hematophagous_bee": "它会从动物身上授粉，目前我还没找到什么办法让它能在JEI里显示。要得到这种蜜蜂，你可以查阅关于它的刷怪蛋的合成表。",
+  "productivebees.ingredient.description.hoarder_bee": "如果你把东西放在地上，这只蜜蜂会把它们收集起来并储存在蜂箱里。可以携带比它的小表亲⸺⸺收集者蜜蜂更多的东西。",
+  "productivebees.ingredient.description.inactive_dragon_egg": "看起来很酷，但少了点什么。试着往上面倒点末影龙的东西。",
+  "productivebees.ingredient.description.jungle_wood_nest": "这个蜂巢将吸引所有主世界生物群系中的蜜蜂。",
+  "productivebees.ingredient.description.leafcutter_bee": "这种蜜蜂比较孤僻，它一般都独自生活在主世界的蜂巢里。查阅关于它的合成表来看看他会在哪些蜂巢中生成。",
+  "productivebees.ingredient.description.lumber_bee": "对原木有着巨大痴迷。",
+  "productivebees.ingredient.description.mason_bee": "这种蜜蜂比较孤僻，它一般都独自生活在主世界的蜂巢里。查阅关于它的合成表来看看他会在哪些蜂巢中生成。",
+  "productivebees.ingredient.description.neon_cuckoo_bee": "这只鬼鬼祟祟的蜜蜂会在蓝色条纹蜜蜂的巢里产卵。",
+  "productivebees.ingredient.description.nether_brick_nest": "吸引下界的熔岩蜜蜂。为了吸引蜜蜂，蜂巢必须放在下界。",
+  "productivebees.ingredient.description.nether_quartz_nest": "吸引下界的下界石英蜜蜂。为了吸引蜜蜂，蜂巢必须放在下界。",
+  "productivebees.ingredient.description.nomad_bee": "这只鬼鬼祟祟的蜜蜂会在灰烬穴居蜜蜂的巢里产卵。",
+  "productivebees.ingredient.description.oak_wood_nest": "这个蜂巢将吸引所有主世界生物群系中的蜜蜂。",
+  "productivebees.ingredient.description.obsidian_nest": "艰苦的生活环境只适合一些末地原生的蜜蜂。",
+  "productivebees.ingredient.description.oily_bee": "这种蜜蜂在陆地上无法找到。如果你想要这只丰满又“油”腻的小蜜蜂，就去深海钓鱼吧。",
+  "productivebees.ingredient.description.only_spawnegg": "要获得这只蜜蜂，请查阅其刷怪蛋的相关合成表。",
+  "productivebees.ingredient.description.pepto_bismol_bee": "治愈你的病痛",
+  "productivebees.ingredient.description.prismarine_bee": "在陆地上找不到这种蜜蜂。渔民们讲述了在深海捕鱼时这只蜜蜂啃食鱼饵的故事。",
+  "productivebees.ingredient.description.quarry_bee": "对石头、沙子和沙砾有着巨大痴迷。",
+  "productivebees.ingredient.description.rancher_bee": "在蜜蜂学校，它的绰号是“布比”。",

```

```suggestion
  "productivebees.ingredient.description.rancher_bee": "在蜜蜂学校，它的绰号是“奶香蜂”。",
```
BooBee音同boobie，指“胸部；乳房”